script/elevate-cpanel.PL

```diff
@@ -552,9 +552,9 @@ sub start ($self) {
         WARN( <<~"EOS");
         Automated upgrade of the distro has been disabled.
         Please wait for the file '$touchfile' to be created.
-        At that stage, the ELevate process is paused, and you can upgrade from 7 to 8 on your own.
+        At that stage, the ELevate process is paused, and you can upgrade the distro on your own.
```

Comment:
When this file is created, the ELevate process is paused and you can upgrade on your own.